src/LogTags.h

```diff
@@ -17,6 +17,9 @@
 class LogTagsErrors
 {
 public:
+    /// constructs an object from an errno produced by Comm::ReadNow()
```

Comment:
I do not think the new method is limited to just ReadNow() context.

```suggestion
    /// constructs an object matching errno(3) of a failed I/O call
```

Reply:
Done (3c246c0).